{
  "original_file": "c:\\Users\\patra\\Genesis FINAL TRY\\modules\\unclassified\\validate_phase_82_83_recovered_2.py",
  "quarantine_timestamp": "2025-06-21T04:33:42.713256+00:00",
  "violations": [
    {
      "rule_type": "no_mock_data",
      "pattern": "test_",
      "file": "c:\\Users\\patra\\Genesis FINAL TRY\\modules\\unclassified\\validate_phase_82_83_recovered_2.py",
      "line_numbers": [
        12,
        109
      ],
      "violation_count": 2
    },
    {
      "rule_type": "no_fallback_logic",
      "pattern": "try:",
      "file": "c:\\Users\\patra\\Genesis FINAL TRY\\modules\\unclassified\\validate_phase_82_83_recovered_2.py",
      "line_numbers": [
        30,
        61,
        84
      ],
      "violation_count": 3
    },
    {
      "rule_type": "no_fallback_logic",
      "pattern": "except Exception",
      "file": "c:\\Users\\patra\\Genesis FINAL TRY\\modules\\unclassified\\validate_phase_82_83_recovered_2.py",
      "line_numbers": [
        56,
        79,
        100
      ],
      "violation_count": 3
    },
    {
      "rule_type": "no_fallback_logic",
      "pattern": "if not",
      "file": "c:\\Users\\patra\\Genesis FINAL TRY\\modules\\unclassified\\validate_phase_82_83_recovered_2.py",
      "line_numbers": [
        25
      ],
      "violation_count": 1
    }
  ],
  "quarantine_reason": "ARCHITECT_MODE_VIOLATION"
}